src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -0,0 +1,12 @@
+import { ChatHeaderStyled, HeaderContent } from './ChatHeaderStyled';
+
+const ChatHeader = ({ countryData }) => (
```

Comment:
Pass the required parameters for a specific component, there is no need to pass the entire object, this will result in additional re-renders.

Reply:
In that case there should be "name" and "participantsAmount"

Reply:
Done